app/assets/javascripts/collections.js

```diff
@@ -0,0 +1,27 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+$(document).ready(function(){
+  var institutionId = 0;
+
+  $('#institution_search').on('click', '.select-option', function(){
+    institutionId = $(this).attr('data-id')
+    loadCollections();
+  });
+
+  function loadCollections(){
+    data = {
+      institution_id: institutionId,
+    };
+    $(".collect-list").html('<div class="loading"></div>')
+    $.ajax({
+        type: "POST",
+        url: "/collections/list",
+        data: {institution_id: institutionId},
+      success: function(data, textStatus, jqXHR){
+      },
```

Comment:
This is pretty poor UX; there's no feedback on whether your action has completed or not.